src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Extensions.Default/DefaultExtensionServiceProvider.cs

```diff
@@ -17,6 +19,10 @@ namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default
     public class DefaultExtensionServiceProvider : IExtensionServiceProvider
     {
         private const string TryConvertProjectConverterStepOptionsSection = "TryConvertProjectConverter";
+        private static readonly string[] TypeMapFiles = new[]
+        {
+            "WebTypeReplacements.typemap"
+        };
```

Comment:
nit: use configuration option so i can say where the file comes from and how many there are. would that be a future enhancement?

Reply:
Yep, good idea. I'll do it as a configuration option

Reply:
I've updated this to use extension configuration options, so users should be able to add their own config settings listing other additional files they want UA to use.